src/app/projects/projects.service.ts

```diff
@@ -62,16 +63,24 @@ export class ProjectsService {
           obj.content.supplementary_links || []
         ),
         publications: obj.publicationsInfo ?? [],
-        authors: obj.content.contributors.map((author) => {
-          const names = author.name.split(',');
-          const formattedName = `${
-            names[names.length - 1]
-          } ${names[0][0].toUpperCase()}`;
+        authors: obj.content.contributors?.map((author) => {
+          let formattedName = '';
+          let fullName = '';
+          if (author.hasOwnProperty('name')) {
+            const names = author.name.split(',');
+            formattedName = `${
+              names[names.length - 1]
+            } ${names[0][0].toUpperCase()}`;
+            fullName = author.name;
+          } else if (author.hasOwnProperty('last')) {
+            formattedName = author.last;
+            fullName = author.last;
+          }
```

Comment:
this is conflicting with https://github.com/ebi-ait/projects-index/pull/57/files

Reply:
yep, that's why it is in Draft ;-)

Reply:
This now fixed